src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -132,6 +148,11 @@ internal sealed class MemberExpansion : Expansion
                 }
             }
 
+            if (dynamicViewExpansion != null)
+            {
+                expansions.Add(dynamicViewExpansion);
+            }
```

Comment:
In the DynamicView case, are we expecting any other expansions? If not, could we simply return dynamicViewExpansion up front, when we determine this is dynamic?